test/export/test_converter.py

```diff
@@ -21,6 +21,26 @@ def forward(self, x, y):
 
         torch.testing.assert_close(ep.module()(*inp)[0], m(*inp))
 
+    def test_convert_if_basic(self):
+        class M(torch.nn.Module):
+            def forward(self, x: torch.Tensor, y: torch.Tensor):
+                if x:
```

Comment:
Non blocking comment. 

In the real model, I see 
```
if x.dtype in (torch.float, torch.half):
    foo
else: 
    bar
````
It will be scripted as `prim::dtype` and `aten::__contains__`. 
In addition, PT2 IR doesn't support dtype being a dynamic value. We should explore how to handle such case.